packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -61,6 +61,42 @@ enum AppUnit {
   }
 }
 
+enum CombinedDiffTypeAppUnit {
+  entireIncrease,
+  entireDecrease,
+  mainCombined,
+  mainIncrease,
+  mainDecrease,
+  deferredCombined,
+  deferredIncrease,
+  deferredDecrease,
+  entireCombined;
+
+  String get display {
```

Comment:
This `display` getter can be removed - we aren't displaying the string anywhere so no need to have the getter